pkg/cmd/discovery/root.go

```diff
@@ -45,13 +47,13 @@ func run() error {
 // and passed to the callback allowing the agent code to access the central config
 func initConfig(centralConfig corecfg.CentralConfig) (interface{}, error) {
 	// default the data plane name
-	if centralConfig.GetDataPlaneName() == "" {
-		centralConfig.SetDataPlaneName("Mulesoft")
+	if centralConfig.GetDataPlaneName() == mulesoft {
+		centralConfig.SetDataPlaneName("")
```

Comment:
Shouldn't this be the other way around?

Reply:
I'm sorry, can you please explain what you mean a bit more?

Reply:
ahh I see now, thanks